packages/ibm-products/src/components/Datagrid/Datagrid/DatagridToolbar.tsx

```diff
@@ -177,14 +190,23 @@ const DatagridBatchActionsToolbar = (
         toolbarBatchActions &&
         toolbarBatchActions?.map((batchAction, index) => {
           if (
-            (index < 2 && toolbarBatchActions.length > 3) ||
-            (index < 3 && toolbarBatchActions.length <= 3)
+            (!toolbarBatchActionsDisplayMin &&
+              index < 2 &&
+              toolbarBatchActions.length > 3) ||
+            (index < 3 && toolbarBatchActions.length <= 3) ||
+            (toolbarBatchActionsDisplayMin !== undefined &&
+              index < toolbarBatchActionsDisplayMin)
           ) {
             return (
               <TableBatchAction
                 key={`${batchAction.label}-${index}`}
                 renderIcon={batchAction.renderIcon}
                 onClick={(event) => onClickHandler(event, batchAction)}
+                className={
```

Comment:
would prefer to see the `classnames` package used here
```
cx({
  ${carbon.prefix}--noLabel: !batchAction.label || batchAction.label === ''
)}
```

Reply:
Sure

Reply:
Fixed